<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="PackageView" name="packageView" native="true">
        <property name="minimumSize">
         <size>
          <width>720</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <widget class="GroupBox" name="groupBox" native="true"/>
      </widget>
      <widget class="InfoTabs" name="infoTabs" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>pakman</string>
    </property>
    <addaction name="actionWhatProvides"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTransaction">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Transaction</string>
    </property>
    <addaction name="actionApply"/>
    <addaction name="actionCancel"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Toolbar"/>
    <addaction name="actionPacman_Log_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDummy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>-</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionSearchByName"/>
    <addaction name="actionSearchByDescription"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTransaction"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDummy"/>
   <addaction name="menuSearch"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSync_Repo"/>
   <addaction name="actionAUR"/>
   <addaction name="actionRefresh_View"/>
   <addaction name="actionSystem_Upgrade"/>
   <addaction name="actionApply"/>
   <addaction name="actionCancel"/>
   <addaction name="actionRoot_Terminal"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/pakman-resource.qrc">
     <normaloff>:/pakman/resource/icons/help.png</normaloff>:/pakman/resource/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help (F1)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSync_Repo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="svn-update">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Sync Repo</string>
   </property>
   <property name="toolTip">
    <string>Sync Repo (Ctrl + Shift + S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRefresh_View">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Refresh View</string>
   </property>
   <property name="toolTip">
    <string>Refresh View (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSystem_Upgrade">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="system-software-update">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>System Upgrade</string>
   </property>
   <property name="toolTip">
    <string>System Upgrade (Ctrl + U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="run-build-install-root">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="dialog-cancel">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionRoot_Terminal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="utilities-terminal">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Root Terminal</string>
   </property>
  </action>
  <action name="actionShow_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
   <property name="shortcut">
    <string>Alt+.</string>
   </property>
  </action>
  <action name="actionSearchByName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By name</string>
   </property>
  </action>
  <action name="actionSearchByDescription">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By description</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWhatProvides">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="system-search">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>What provides me ...</string>
   </property>
  </action>
  <action name="actionPacman_Log_Viewer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pacman Log Viewer</string>
   </property>
  </action>
  <action name="actionAUR">
   <property name="text">
    <string>AUR</string>
   </property>
   <property name="toolTip">
    <string>Sync AUR</string>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionInstall_now">
   <property name="text">
    <string>Install now</string>
   </property>
  </action>
  <action name="actionRemove_now">
   <property name="text">
    <string>Remove now</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PackageView</class>
   <extends>QWidget</extends>
   <header>src/ui/packageview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QWidget</extends>
   <header>src/ui/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoTabs</class>
   <extends>QWidget</extends>
   <header>src/ui/infotabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/pakman-resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShow_Toolbar</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
